package.json

```diff
@@ -1409,6 +1409,11 @@
             "type": "object",
             "required": [],
             "properties": {
+              "processCommand": {
```

Comment:
>processCommand [](start = 15, length = 14)

I believe you also need a tweak [here](https://github.com/OmniSharp/omnisharp-vscode/blob/4a26dce82e038a76bb78ca2870a9fb1ec3af97e5/src/tools/GenerateOptionsSchema.ts#L119) to delete `processCommand` from the unit test options.

Reply:
Done.